sdk/appconfiguration/app-configuration/README.md

```diff
@@ -1,114 +1,108 @@
 
 ## Getting started
 
 ### Currently supported environments
 
-- Node.js version 6.x.x or higher
-- Browser JavaScript
+- Node.js version 8.x.x or higher
 
 ### How to Install
 
 ```bash
-npm install @azure/app-config
+npm install @azure/app-configuration
 ```
+
 ## Key concepts
 
-### How to use
+### Configuration Setting
 
-#### nodejs - Authentication, client creation and listConfigurationSettings  as an example written in TypeScript.
+A Configuration Setting is the fundamental resource within a Configuration Store.
+In its simplest form, it is a key and a value. However, there are additional properties such as 
+the modifiable content type and tags fields that allows the value to be interpreted or associated 
+in different ways.
 
-##### Install @azure/ms-rest-nodeauth
+The `label` property of a Configuration Setting provides a way to separate configuration settings 
+into different dimensions. These dimensions are user defined and can take any form. Some common 
+examples of dimensions to use for a label include regions, semantic versions, or environments. 
+Many applications have a required set of configuration keys that have varying values as the 
+application exists across different dimensions.
 
-```bash
-npm install @azure/ms-rest-nodeauth
-```
+For example, MaxRequests may be 100 in "NorthAmerica", and 200 in "WestEurope". By creating a 
+Configuration Setting named MaxRequests with a label of "NorthAmerica" and another, only with 
+a different value, in the "WestEurope" label, an application can seamlessly retrieve 
+Configuration Settings as it runs in these two dimensions.
 
 ## Examples
 
+#### nodejs - Authentication, client creation and listConfigurationSettings as an example written in TypeScript.
+
 ##### Sample code
 
 ```typescript
-import * as coreHttp from "@azure/core-http";
-import * as coreArm from "@azure/core-arm";
-import * as msRestNodeAuth from "@azure/ms-rest-nodeauth";
-import { ConfigurationClient, ConfigurationModels, ConfigurationMappers } from "@azure/app-config";
-const subscriptionId = process.env["AZURE_SUBSCRIPTION_ID"];
-
-msRestNodeAuth.interactiveLogin().then((creds) => {
-  const client = new ConfigurationClient(creds, subscriptionId);
-  const label = ["testlabel"];
-  const key = ["testkey"];
-  const acceptDateTime = new Date().toISOString();
-  const fields = ["etag"];
-  client.listConfigurationSettings(label, key, acceptDateTime, fields).then((result) => {
-    console.log("The result is:");
-    console.log(result);
-  });
-}).catch((err) => {
-  console.error(err);
+import { AppConfigurationClient } from "@azure/app-configuration";
+
+const connectionString = process.env["AZ_CONFIG_CONNECTION"]!;
+const client = new AppConfigurationClient(connectionString);
+
+const label = ["testlabel"];
+const key = ["testkey"];
+const acceptDateTime = new Date();
+const fields = ["etag"];
+
+await client.listConfigurationSettings({
```

Comment:
Since you're using await, you can rewrite this as
```typescript
const result = await client.listConfigurationSettings({
  label,
  key,
  acceptedDateTime,
  fields
});

console.log("The result is:", result);
```
You should also give an explanation of what some of the fields are...just looking at it I'm not sure what `acceptDateTime` is doing, or what specifying `fields` will do either.

Nit: I don't think top-level await exists yet, you'd need to use await inside of an async function.

Reply:
I modified it to e a simpler API that basically does the same thing. The listConfigurationSetting() api is a bit more complex and could use a better sample which I can add in another PR.